src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/BeforeActions.java

```diff
@@ -31,4 +31,10 @@ public void loginIfRequired() throws IOException {
     logger.info("-----------------Logging in if required------------------");
     Helper.loginIfRequired();
   }
+
+  @Before(order = 0)
+  public void createTmpDirectory() throws IOException {
```

Comment:
This method creates a tmp dir in the runner with files in the fixtures that will be used by the pipelines